corehq/apps/reports/tasks.py

```diff
@@ -165,12 +165,11 @@ def is_app_active(app_id, domain):
 
 @periodic_task(run_every=crontab(hour="2", minute="0", day_of_week="*"), queue='background_queue')
 def apps_update_calculated_properties():
-    q = {"filter": {"and": [{"missing": {"field": "copy_of"}}]}}
-    results = stream_es_query(q=q, es_index='apps', size=999999, chunksize=500)
+    results = AppES().is_build(False).source(['_id', 'domain']).scroll()
```

Comment:
There is no test case for this, but it's a pretty trivial change. I tested this on local data to make sure it's correct.